client/src/annoMatrix/annoMatrix.ts

```diff
@@ -631,48 +632,49 @@ Return cache keys for columns associated with this query.  May return
       //   )}]`
       // );
       this._cache[field] = toDrop.reduce(
-        // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'df' implicitly has an 'any' type.
-        (df, col) => df.dropCol(col),
+        (df: Dataframe, col: string) => df.dropCol(col),
```

Comment:
Wrong DataFrame type - should be instance of class DataFrame (util/dataframe), not the schema sub-object